src/Microsoft.Health.Fhir.Liquid.Converter/Models/Json/JsonContentDocument.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using DotLiquid;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Models.Json
+{
+    public class JsonContentDocument : Document
```

Comment:
what's the difference between document and  JsonContentDocument?

Reply:
Renamed to ```JSchemaDocument```, and added a ```JSchema Schema``` member in ```JSchemaDocument```